node/bft/src/gateway.rs

```diff
@@ -175,7 +175,7 @@ impl<N: Network> Gateway<N> {
         worker_senders: IndexMap<u8, WorkerSender<N>>,
         sync_sender: Option<SyncSender<N>>,
     ) {
-        debug!("Starting the gateway for the memory pool...");
+        trace!("Starting the gateway for the memory pool...");
```

Comment:
```suggestion
        info!("Starting the gateway for the memory pool...");
```

This should be an `info` log like all the other module bootup messages.